cves/kernel/CVE-2019-15216.yml

```diff
@@ -106,7 +101,7 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 6bc235a2e24a5ef677daee3fd4f74f6cd643e23c
-  note: Discovered automatically by archeogit.
+  note: Manually verified
 upvotes_instructions: |
```

Comment:
This vulnerability actually seems pretty interesting given its about USB ports and existed for so long. I'll give it a 6